modules/nf-core/cobs/classicconstruct/main.nf

```diff
@@ -0,0 +1,52 @@
+process COBS_CLASSICCONSTRUCT {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/cobs:0.3.0--hdcf5f25_1' :
+        'biocontainers/cobs:0.3.0--hdcf5f25_1'}"
+
+    input:
+    tuple val(meta), path(input)
+
+    output:
+    tuple val(meta), path("${index}"), emit: index
+    path "versions.yml"              , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    index = "${prefix}.index.cobs_classic"
+    def task_memory_in_bytes = task.memory.toBytes()
+    """
+    cobs \\
+        classic-construct \\
+        $args \\
+        --memory $task_memory_in_bytes \\
+        --threads $task.cpus \\
+        $input \\
+        $index
```

Comment:
Maybe just use `${prefix}.index.cobs_classic` instead of creating a new variable? Just seems to be more standard.

Reply:
Yeah, it was like this before, but then I had the need to copy `${prefix}.index.cobs_classic` to the `output` section of the module as well ([here](https://github.com/nf-core/modules/pull/4586/files#diff-e8d7b3a1a8df4da5806ce102df51031d9e995f7edb9530648466c5da4cede602R14)). Instead of having two copies of these, I refactored into a new variable `index` and inserted this variable in both places. I am happy with any of these options, so I prefer to leave to you the decision of keeping or removing this `index` variable. Please let me know your decision